index.html

```diff
@@ -1,14 +1,339 @@
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div>
+        <div class="chef">
+          <div class="image-container">
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div>
+        <div class="chef">
+          <div class="image-container">
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div>
+        <div class="chef">
+          <div class="image-container">
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div> -->
+      </div>
+    </section>
+
+    <section class="partner">
+      <div class="section-title partner-title">
+        <p>Partner</p>
+        <hr />
+      </div>
+      <div class="partners">
+        <img src="./Assets/micro.png" class="micro" alt="Microverse" />
+        <img src="./Assets/pizzahot.png" class="pizza" alt="pizzahot" />
+        <img src="./Assets/mar.png" class="mar" alt="Moroccan kingdom logo" />
+        <img src="./Assets/prt.png" class="prt" alt="top chef">
+        <img src="./Assets/mchef.png" class="master" alt="Master chef logo" />
+      </div>
+    </section>
+    <footer>
+      <a class="logo-container" href="#">
+        <img src="./Assets/logo.svg" alt="Logo Icon" />
+        <h3>The Moroccan Kitchen.</h3>
+      </a>
+      <p>
+        Created by
+        <a
+          rel="noopener"
+          href="https://www.linkedin.com/in/otmane-echchafyky-125801248/"
+          target="_blank"
+          >OTMANE.</a
+        ><br />
+        Designed by Cindy Shin
+      </p>
+    </footer>
```

Comment:
from the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), your index page on mobile should end with the `more` button and not the partner and footer section😉 Kindly study the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and make the necessary change👍
<img width="313" alt="image" src="https://user-images.githubusercontent.com/64643995/233625871-3eb73677-1d17-4c95-b9ac-1f26f455517e.png">

Don't forget we are required to stick to the given design in this milestone even as we personalize it😉